test/functional-tests/include/Exception.hpp

```diff
@@ -39,7 +39,13 @@ namespace parameterFramework
 class Exception : public std::runtime_error
```

Comment:
All the exceptions from this module (functional-tests) throw this exception. It was an attempt to distinguish exceptions thrown by the test functions (expected) from unexpected exception from unknown source.

But I am no longer convinced that this is a good idea though.